unitTests/ui/ResponsiveComponents/ResponsiveFacetsTest.ts

```diff
@@ -219,5 +220,15 @@ export function ResponsiveFacetsTest() {
 
       expect(responsiveFacets.needDropdownWrapper()).toBe(false);
     });
+
+    it('should use the breakpoint set from the search interface if not specified explicitely', () => {
```

Comment:
Would it be useful to include a test where the breakpoint value is set globally as well as on the facet, and checking that the facet value is used?

Unless we want the global to override, in which case the same test, but checking that the global value is used.

Reply:
Yeah good idea. 👍